<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsPage</class>
 <widget class="QWidget" name="GeneralSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>455</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="apiKeyLabel">
     <property name="text">
      <string>API Key:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="apiKeyEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="apiKeyHelpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>You can find it in the &quot;API access&quot; tab in your store settings.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/res/icons/information.png</normaloff>:/res/icons/information.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="targetCurrencyLabel">
     <property name="text">
      <string>Target Currency:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="targetCurrencyCombo"/>
     </item>
     <item>
      <widget class="QPushButton" name="targetCurrencyHelpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>If your Stripe is in a different currency than EUR, you can select it here. Otherwise select EUR.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res.qrc">
         <normaloff>:/res/icons/information.png</normaloff>:/res/icons/information.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="behaviorGroupBox">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="closeToTrayCheckBox">
        <property name="text">
         <string>Close Lectronizer to system tray</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="autoFetchCheckBox">
        <property name="text">
         <string>Automatically fetch new orders when minimized</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="autoFetchIntervalLabel">
        <property name="text">
         <string>Fetch every:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="autoFetchIntervalSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>1440</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="shippingGroupBox">
     <property name="title">
      <string>Shipping</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="shippingTrackingUrlLabel">
        <property name="text">
         <string>Tracking URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="shippingTrackingUrlEdit">
          <property name="placeholderText">
           <string>https://example.com/tracking/{#}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shippingTrackingUrlHelpButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Use &quot;{#}&quot; to mark where the tracking ID should be inserted.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../res.qrc">
            <normaloff>:/res/icons/information.png</normaloff>:/res/icons/information.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="displayGroupBox">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="friendlyDateCheckBox">
        <property name="text">
         <string>Use friendly date format (&quot;Today&quot;, &quot;Yesterday&quot;, etc.)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="csvExportGroupBox">
     <property name="title">
      <string>CSV Export</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="csvSeparatorLabel">
        <property name="text">
         <string>Separator:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="csvSeparatorComboBox">
        <item>
         <property name="text">
          <string>Comma ,</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Semicolon ;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tab \t</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
